docs/web_reference.rst

```diff
@@ -1128,6 +1135,13 @@ duplicated like one using :meth:`Application.copy`.
       :param kwargs: additional parameters for :class:`RequestHandlerFactory`
                      constructor.
 
+      .. note::
```

Comment:
Ok, done.

![image](https://cloud.githubusercontent.com/assets/957767/18017881/6e73a534-6bdc-11e6-8475-342b909e3fb1.png)

I just doubt if this could confuse user and he could briefly read this as

```
kwargs - additional parameters for RequestHandlerFactory constructor.

    deprecated since 1.0: (blah-blah tl;dr!)
```
and he thinks "ok probably those kwargs are deprecated".